assets/api-swagger/swagger.json

```diff
@@ -914,6 +914,228 @@
               }
             }
           }
+        }
+      },
+      "get": {
+        "tags": [
+          "Profiles"
+        ],
+        "summary": "Get the status of all merge jobs in a given account.",
+        "description": "Get the status of all merge jobs in a given account.",
+        "parameters": [
+          {
+            "name": "Ometria-Account-Id",
```

Comment:
this does not seem correct, i don't think they send account id, my understanding is they send api key and that gets swapped out for accounts id somewhere along the api gateway (i want to say the auth webhook)

Reply:
Yes, you're right. I copied it from the inner endpoint which needs it but here it can be removed. Thanks